aci/resource_aci_bgpprotp.go

```diff
@@ -138,49 +162,67 @@ func resourceAciL3outBGPProtocolProfileCreate(ctx context.Context, d *schema.Res
 	}
 	d.Partial(false)
 
+	if relationTobgpRsBestPathCtrlPol, ok := d.GetOk("relation_bgp_rs_best_path_ctrl_pol"); ok {
+		relationParam := relationTobgpRsBestPathCtrlPol.(string)
```

Comment:
Maybe put the GetMoName() here like the next function do.

Reply:
removed